### PR TITLE
version changes required to work with vuepress-theme-hope 2.0.0-rc.59

### PR DESCRIPTION
**Pull request by Amndeep7**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,16 +16,15 @@
   },
   "devDependencies": {
     "@vuepress/bundler-vite": "2.0.0-rc.18",
-    "@vuepress/bundler-webpack": "2.0.0-rc.17",
```
Comment (Amndeep7) [inline package.json:19]: Forgot to make this a separate commit, but I got rid of the webpack bundler since we were using vite to build the site and removing it didn't seem to break anything that I saw.

**Comment by Amndeep7** [comment]
For the reviewer's reference: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#groups

**Review by wdower — APPROVED** [review]
Looks good. Site still builds, and the markdown tabs etc still work.

**Review by georgedias — COMMENTED** [review]
Please review my comments, not necessary required modification, just observations.

File: src/courses/advanced/11.md [hunk 2]
```diff
@@ -265,11 +265,11 @@ From here, we can download that file and manually drop it off in something like
...
+Heimdall is a very powerfull tool for examining security data. See more details in the [SAF User Class](../user/09.md)
 :::
 
 ### What Else Can We Do With A Pipeline?
 
 In a real use case, if our pipeline passed, we would next save our bonafide hardened image to a secure registry where it could be distributed to users. If the pipeline did not pass, we would have already collected data describing why, in the form of InSpec scan reports that we save as artifacts.
 
-Other pipelines might flat out refuse to permit merging code into a repository branch, or alert the developer team if issues occur. The exact implementation is up to you and your needs for your work.
\ No newline at end of file
+Other pipelines might flat out refuse to permit merging code into a repository branch, or alert the developer team if issues occur. The exact implementation is up to you and your needs for your work.
```
[the remark this answers is not in the document]
Comment (Amndeep7) [reply src/courses/advanced/11.md:275]: I think my editor got rid of the new line

File: .github/dependabot.yml [hunk 3]
```diff
@@ -10,4 +10,10 @@ updates:
     # Check the npm registry for updates every day (weekdays)
```
[the remark this answers is not in the document]
Comment (Amndeep7) [reply .github/dependabot.yml:10]: The version and update attributes were just elided from the diff but they're up there.

I don't think it's as important to set the exact time we want updates here as it was in the monorepo (where I was trying to make what used to be the gajillion dependabot PRs going off at the same time making it impossible to have any of the workflows run against a PR that was being developed)
